packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.14.0] - 2021-05-19
+### Added
+- refactor api.rpc, only expose historic RPC and set to the current block hash. (#304)
+
+### Fixed
+- fetch service failed to get the latest block while API connection is on, but subscription to finalized head is dropped. 
```

Comment:
too complex to me.
`use pull instead of subscribe to get new block height, in case subscription stalled`